### PR TITLE
Add support for MathWallet extension

### PR DESCRIPTION
**Pull request by yuzhiyou1990**

### CODE REVIEW

**Comment by ericfish** [comment]
Change submitted by MathWallet (https://mathwallet.org)

Detail changes:
1. add MathWalletAdapter code (ref SolongWalletAdapter)
2. add math/index.tsx (ref SolongWalletAdapter basiclly the same)

Contact: ericyu@mathwallet.org

File: src/wallet-adapters/math/index.tsx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+      notify({
+        message: 'Math Wallet Error',
+        description: 'Please install mathwallet',
+      });
+      return;
+    }
+
+    this._onProcess = true;
+    this._provider
+      .getAccount()
```
Comment (nathanielparke) [inline src/wallet-adapters/math/index.tsx:70]: Getting an error when calling `getAccount`, is this supposed to be `getAccounts()`?

**Review by nathanielparke — CHANGES_REQUESTED** [review]
Looks good. Will merge once issue with `getAccount()` is resolved.

File: src/wallet-adapters/math/index.tsx (hunk 1 above)
Comment (yuzhiyou1990) [reply src/wallet-adapters/math/index.tsx:70]: image
The getAccount method is correct, update MathWallet extension to v2.6.4.

Comment (nathanielparke) [reply src/wallet-adapters/math/index.tsx:70]: Agreed v2.6.4 works

**Pull request merged**